src/full/Agda/Compiler/MAlonzo/Compiler.hs

```diff
@@ -212,15 +220,14 @@ imports = (hsImps ++) <$> imps where
               T.PAdd64, T.PSub64, T.PMul64, T.PQuot64, T.PRem64, T.PLt64, T.PEq64,
               T.PITo64, T.P64ToI] -- Excludes T.PEqF, which is defined in MAlonzo.RTE.Float
 
-  imps :: TCM [HS.ImportDecl]
-  imps = List.map decl . uniq <$>
-           ((++) <$> importsForPrim <*> (List.map mazMod <$> mnames))
+  imps :: [HS.ImportDecl]
+  imps = decl <$> (uniq $ (importsForPrim builtinThings defs) ++ (mazMod <$> mnames))
```

Comment:
For lists, I really prefer `map` over `<$>`.
```haskell
imps = map decl $ uniq $ importsForPrim builtinThings defs ++ map mazMod mnames
```
Yeah, all parentheses gone!

Reply:
Will do

Reply:
(Done!)